Makefile

```diff
@@ -1,13 +1,12 @@
+CPUS ?= $(shell sysctl -n hw.ncpu || echo 1)
```

Comment:
yeah we could probably tweak it, but i was mostly focused on making it faster in dev